evap/evaluation/templates/choice_button.html

```diff
@@ -2,10 +2,10 @@
 
 <label class="btn btn-sm btn-option
     {% if formelement.field.disabled or disabled %} disabled{% endif %}
-    {% if formelement.value == choice.data.value %} active{% endif %}
+    {% if choice.data.selected %} active{% endif %}
     {% if formelement.errors %} choice-error{% endif %}"
-    name="{{ choice.data.name }}" data-toggle="tooltip" data-placement="top" title="{{ tooltip }}">
```

Comment:
why do you remove the name?

Reply:
It was part of a `<label>` and a `name` attribute wasn’t sensible for me. The `name` is on the contained checkbox and the reference between label and checkbox is provided through nesting (in other cases, one would use `id` and `for`, but not `name`)

Reply:
i was just wondering why it was there in the first place but it can probably removed.